src/components/common/NameRequestInfo.vue

```diff
@@ -24,15 +24,16 @@
               <li id="name-request-title">
                 <strong>{{ getNameRequestNumber }}</strong> - {{ getNameRequestApprovedName }}
               </li>
-              <li class="mt-4">
+
+              <li class="bullet-point mt-4">
                 <strong>Entity Type:</strong> {{ getEntityTypeDescription }}
               </li>
-              <li><strong>Request Type:</strong> {{ requestType }}</li>
-              <li><strong>Expiry Date:</strong> {{ expirationDate }}</li>
-              <li><strong>Status:</strong> {{ state }}</li>
+              <li class="bullet-point mt-4"><strong>Request Type:</strong> {{ requestType }}</li>
```

Comment:
I see a handful of GitHubActions (lint) warnings below. Can you fix please?

Reply:
I updated this part and omitted the changes I've made with the latest push